app/templates/briefs/check_your_answers.html

```diff
@@ -31,12 +31,9 @@
 {% block main_content %}
 <div class="grid-row">
   <div class="column-two-thirds">
-    {% with
-      heading = "Check and submit your answers" if brief_response.status == 'draft' else "Your application for ‘{}’".format(brief.title),
-      smaller = true
-      %}
-        {% include 'toolkit/page-heading.html' %}
-    {% endwith %}
+    <h1 class="govuk-heading-l">
+      {{ "Check and submit your answers" if brief_response.status == 'draft' else "Your application for ‘{}’".format(brief.title) }}
```

Comment:
I'm finding this a bit hard to read, could we improve this?

```suggestion
	  {% if brief_response.status == 'draft' %}
      	Check and submit your answers
      {% else %} 
      	Your application for {{ brief.title }} 
      {% endif %}
```